test_general.py

```diff
@@ -35,3 +35,11 @@ def test_link_with_target():
     expected = "Please click <http://xxx.com/t.html|here>"
     output = HTMLSlacker(html).get_output()
     assert(output == expected)
+
+def test_unordered_list():
+    html = '<ul><li>Hello</li></ul>'
+    expected = "- Hello"
```

Comment:
It is necessary to append
```
    output = HTMLSlacker(html).get_output()
    assert (output == expected)
```

Otherwise the test will not work.